pkgs/desktops/deepin/go-package/go-lib/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, replaceAll
+, glib
+, xorg
+, gdk-pixbuf
+, pulseaudio
+, mobile-broadband-provider-info
+, runtimeShell
+}:
+
+stdenv.mkDerivation rec {
+  pname = "go-lib";
```

Comment:
This is a go-module. Why do we need to package this? Other go programs and libraries would likely consume it through go modules, right?

Reply:
gomod support add on https://github.com/linuxdeepin/go-lib/pull/70， I will try to upgrade to 5.9.1

Reply:
In fact, go-lib does not support go modules, this project has the following files：
backlight/keyboard/testdata/input7::capslock/
backlight/keyboard/testdata/xxx::kbd_backlight/

“::” in file name is illegal for go

Reply:
Their go usage is ... interesting

Reply:
deepin is a debian-based distribution, and the deb packaging system does not support go mod perfectly
deepin developers basically do not use go mod to manage dependencies，we can find `export GO111MODULE=off` in Makefile
It looks like [Decodetalkers](https://github.com/Decodetalkers)  is working on this https://github.com/linuxdeepin/go-lib/pull/83

Reply:
As long as we don't need to patch to much around it. Otherwise we could just create a go.mod and use that.

Reply:
I tend to wait for Decodetalkers 's work,  deepin v23 is a major version update and needs a lot of work to port